### ISSUE TITLE
Choroid thickening

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10903]
Preferred term label:
Choroid thickening

Synonyms
Pachychoroid, Thickened choroid, Increased choroidal thickness

Definition (free text, please give PubMed ID)
A condition characterized by an abnormal increase in the thickness of the choroid, which may be associated with various ocular or systemic conditions. Choroid thickening can be measured using imaging techniques such as optical coherence tomography (OCT) (PMID: 39095470)

Parent term (use hpo.jax.org/app)
Abnormal choroid morphology (HP:610)

Diseases characterized by this term
Central serous chorioretinopathy

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001181
- label: Choroid thickening
- parents: [Abnormal choroid morphology (HP:0000610)](https://hpo.jax.org/app/browse/term/HP:0000610)
- pmid: PMID:39095470
- def: A condition characterized by an abnormal increase in the thickness of the choroid, which may be associated with various ocular or systemic conditions. Choroid thickening can be measured using imaging techniques such as optical coherence tomography (OCT).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10903
- synonyms: Increased choroidal thickness(NONE)|Pachychoroid(NONE)|Thickened choroid(NONE)
